diff --git a/front_end/panels/console/ConsolePinPane.ts b/front_end/panels/console/ConsolePinPane.ts
index 705f6f4095..4ca4d5aaa3 100644
--- a/front_end/panels/console/ConsolePinPane.ts
+++ b/front_end/panels/console/ConsolePinPane.ts
@@ -299,7 +299,7 @@ export class ConsolePinPresenter extends UI.Widget.Widget {
     this.#view = view;
 
     this.#selfXssWarningDisabledSetting = Common.Settings.Settings.instance().createSetting(
-        'disable-self-xss-warning', false, Common.Settings.SettingStorageType.SYNCED);
+        'disable-self-xss-warning', true, Common.Settings.SettingStorageType.SYNCED);
 
     this.#pinEditor = {
       workingCopy: () => this.#editor?.state.doc.toString() ?? '',
diff --git a/front_end/panels/console/ConsolePrompt.ts b/front_end/panels/console/ConsolePrompt.ts
index 6667a47803..10d7db3718 100644
--- a/front_end/panels/console/ConsolePrompt.ts
+++ b/front_end/panels/console/ConsolePrompt.ts
@@ -389,7 +389,7 @@ export class ConsolePrompt extends ConsolePromptBase {
         scrollIntoView: true,
       });
       Common.Settings.Settings.instance()
-          .createSetting('disable-self-xss-warning', false, Common.Settings.SettingStorageType.SYNCED)
+          .createSetting('disable-self-xss-warning', true, Common.Settings.SettingStorageType.SYNCED)
           .set(true);
       this.#selfXssWarningShown = false;
       Host.userMetrics.actionTaken(Host.UserMetrics.Action.SelfXssAllowPastingInConsole);
diff --git a/front_end/panels/console/ConsoleView.ts b/front_end/panels/console/ConsoleView.ts
index 94203451d6..056ae30159 100644
--- a/front_end/panels/console/ConsoleView.ts
+++ b/front_end/panels/console/ConsoleView.ts
@@ -490,7 +490,7 @@ export class ConsoleView extends UI.Widget.VBox implements
     this.consoleHistoryAutocompleteSetting = Common.Settings.Settings.instance().resolve(
         Settings.ConsoleSettings.consoleHistoryAutocompleteSettingDescriptor);
     this.selfXssWarningDisabledSetting = Common.Settings.Settings.instance().createSetting(
-        'disable-self-xss-warning', false, Common.Settings.SettingStorageType.SYNCED);
+        'disable-self-xss-warning', true, Common.Settings.SettingStorageType.SYNCED);
 
     const settingsPane = this.contentsElement.createChild('div', 'console-settings-pane');
     UI.ARIAUtils.setLabel(settingsPane, i18nString(UIStrings.consoleSettings));
